tests/framework/extensions/customresourcedefinitions/delete.go

```diff
@@ -0,0 +1,45 @@
+package customresourcedefinitions
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// deletes a single custom resource definition by name
+func DeleteCustomResourceDefinition(client *rancher.Client, clusterID string, namespace string, name string) error {
+	dynamicClient, err := client.GetDownStreamClusterClient(clusterID)
+	if err != nil {
+		return err
+	}
+
+	customResourceDefinitionResource := dynamicClient.Resource(CustomResourceDefinitions).Namespace(namespace)
+	_, err = customResourceDefinitionResource.List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
```

Comment:
Is this list and error check required? It seems like the return value isn't used, could you please remove this block if this is the case?

Reply:
removed unnecessary error check